### PR TITLE
Introduced issue template

### PR DESCRIPTION
**Pull request by abdallahbeshi**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
```
Comment (KaliCZ) [inline .github/ISSUE_TEMPLATE/bug_report.md:7]: Can we remove empty line?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+---
+
+**Describe the bug**
```
Comment (KaliCZ) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]: Maybe I'd structure it just like our youtrack issues - so the person reads the sectiona header, removes the body and writes his own body. So let's put something, so the user doesn't need to write a header

```suggestion
## Bug description
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```
Comment (KaliCZ) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]:
```suggestion
## Steps To Reproduce
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+Steps to reproduce the behavior.
+
+**Expected behavior**
```
Comment (KaliCZ) [inline .github/ISSUE_TEMPLATE/bug_report.md:16]:
```suggestion
## Expected behavior
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
```
Comment (KaliCZ) [inline .github/ISSUE_TEMPLATE/bug_report.md:19]:
```suggestion
## Screenshots or other proof
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+If applicable, add screenshots to help explain your problem.
+
+**Additional context**
```
Comment (KaliCZ) [inline .github/ISSUE_TEMPLATE/bug_report.md:22]:
```suggestion
## Additional context
```

**Review by KaliCZ — COMMENTED** [review]
If we can use headers that is

**Review by KaliCZ — APPROVED** [review]

**Review by marektresnak — APPROVED** [review]

**Pull request merged**